### PR TITLE
Add tables and table API

### PR DESCRIPTION
**Pull request by lnjX**
Here is a table API. (I only wanted to add one wooden table, but then I thought that there are fences, ... in all variants of wood too.)
The API is like the one from the fences.
I hope that aren't too much nodes for the Creative Inv.. But I think the tables are important, because you can put things on them (not like the old fence + pressure plate / carpet variant)!
There are also stone / cobble tables!

Here is a small screenshot!
screenshot_20160125_214832

^^ And I made a small bug fix in logs.lua, hope that's ok in this pull request.

And the crafting recipes:
MAT  MAT  MAT
-----   MAT  -----
-----   MAT  -----
MAT -> material (wood / stone / cobble)

### CODE REVIEW

**Comment by BlockMen** [comment]
It seems like stone tables are fuel currently, no?

**Comment by lnjX** [comment]
:arrow_up:   Fixed that.
And does stone tables should drop cobble ones?

EDIT:
Also the stone tables are at the moment cracky=2 the normal stone blocks have got cracky=3. Is that ok? - I think its a table and not a complete block, so it should be digged faster

File: mods/default/nodes/tables.lua [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	-- set defaults if nothing is set already
+	if not def.groups then
+		def.groups = {choppy = 2, flammable = 2, oddly_breakable_by_hand = 2, fuel = 6}
+	end
+	if not def.sounds then
+		def.sounds = default.node_sound_wood_defaults()
+	end
+
+	-- set table properties
+	def.tiles = {def.texture}
```
Comment (BlockMen) [inline mods/default/nodes/tables.lua]: Is there a specific reason to allow only one texture and not all possible 6 tiles?

Comment (lnjX) [inline mods/default/nodes/tables.lua]: No, I'll change that. :smile_cat:

**Comment by lnjX** [comment]
I merged that in my fork CMN - Continued Minetest NeXt
https://github.com/gamerbude/minetest_next

**Pull request closed without merging**